scripts/verify_included_modules.py

```diff
@@ -150,4 +156,5 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    _, build_root = sys.argv
```

Comment:
I can add support for it. I wanted to manually "enforce" the argument without too much extra code.